fec.config.js

```diff
@@ -1,13 +1,122 @@
+const { resolve } = require('path');
+const packageJson = require('./package.json');
+const webpack = require('webpack');
+
+const bundle = 'insights';
+const appName = packageJson[bundle].appname;
+
 module.exports = {
-  appUrl: '/staging/starter',
-  debug: true,
-  useProxy: true,
-  proxyVerbose: true,
-  /**
-   * Change to false after your app is registered in configuration files
-   */
+  appName,
+  appUrl: `/${bundle}/${appName}`,
+  useProxy: process.env.PROXY === 'true',
   interceptChromeConfig: false,
-  /**
-   * Add additional webpack plugins
-   */
+  debug: true,
+  plugins: [
+    new webpack.DefinePlugin({
+      IS_DEV: false,
+    }),
+  ],
+  moduleFederation: {
+    shared: [
+      {
+        'react-router-dom': {
+          singleton: true,
+          import: false,
+          version: packageJson.dependencies['react-router-dom'],
+          requiredVersion: '>=6.0.0 <7.0.0',
+        },
+      },
+    ],
+    root: resolve(__dirname, './'),
```

Comment:
Is this really needed?

Reply:
its been there for quite some time, I kept it simply because I wasnt confident it wouldnt affect another app

Reply:
I'd try to remove it, so we keep the config at really the bare minimum.

Reply:
While testing the PR I tried running it with `root` and it appeared to work fine.

Reply:
Oh, I just saw that I mean "without". 🙈

Meaning we should remove this `root` prop.